common/_static/disqus.js

```diff
@@ -0,0 +1,20 @@
+$(function(){
+  var button = $('#showComments');
+  
+  if (button.length)
+  {
+    var handler = function(){
+      /* * * CONFIGURATION VARIABLES: EDIT BEFORE PASTING INTO YOUR WEBPAGE * * */
+      var disqus_shortname = 'aspnetdocs'; // required: replace example with your forum shortname
+
+      /* * * DON'T EDIT BELOW THIS LINE * * */
+      var dsq = document.createElement('script'); dsq.type = 'text/javascript'; dsq.async = true;
+      dsq.src = 'https://' + disqus_shortname + '.disqus.com/embed.js';
+      (document.getElementsByTagName('head')[0] || document.getElementsByTagName('body')[0]).appendChild(dsq);
```

Comment:
You've used a JQuery style DOM selector in line 1 and a vanilla js DOM selector in line 13. Maybe try and be consistent?